contrib/nydusify/pkg/converter/converter.go

```diff
@@ -65,7 +65,11 @@ func Convert(ctx context.Context, opt Opt) error {
 			}
 			// We should only clean up when the work directory not exists
 			// before, otherwise it may delete user data by mistake.
-			defer os.RemoveAll(opt.WorkDir)
+			// If the user wants to save the metrics collected during conversion,
+			// we should not clear the work directory.
+			if !opt.OutputJSON {
```

Comment:
Maybe we should allow users to specify the file path like `nydus-image create --output-json ...`?

Reply:
> Maybe we should allow users to specify the file path like `nydus-image create --output-json ...`?

@imeoer  Replace the work directory with a user-specified directory?

Reply:
let output-json be a string arg?

Reply:
fixed